python/tvm/relay/backend/contrib/ethosu/codegen.py

```diff
@@ -31,6 +31,12 @@
 from tvm.relay.backend.contrib.ethosu.op import op_attrs
 from tvm.relay.backend.contrib.ethosu import op
 
+# We are currently using copy_constants scheduler In the long run,
+# this should be a single intelligent and a composite scheduler
+# that can perform scheduling based on user inputs such as
+# scratch memory size.
+CASCADER = copy_constants()
```

Comment:
nit : let us use something like SCHEDULER -- because cascading is a way of scheduling and copy_constants is not a cascader scheduler.

Reply:
Done